### PR TITLE
Update pypi.python.org URLs to pypi.org

### PR DESCRIPTION
**Pull request by jdufresne**
For details on the new PyPI, see the blog post:

https://pythoninsider.blogspot.ca/2018/04/new-pypi-launched-legacy-pypi-shutting.html

### CODE REVIEW

**Comment by jdufresne** [comment]
Thanks, Done!

File: test/test_proxymanager.py [hunk 1]
```diff
@@ -5,12 +5,12 @@
 
 class TestProxyManager(object):
     def test_proxy_headers(self):
-        url = 'http://pypi.python.org/test'
+        url = 'https://pypi.org/project/urllib3/'
```
Comment (SethMichaelLarson) [inline test/test_proxymanager.py]: Sorry for the nit but could you turn this back to `http`? I want to maintain the same code path being tested. :)

Comment (jdufresne) [reply test/test_proxymanager.py]: No problem. Done in the latest version.

**Comment by SethMichaelLarson** [comment]
Thank you @jdufresne! :tada:

**Pull request merged**